src/reference/asciidoc/whats-new.adoc

```diff
@@ -16,17 +16,22 @@ You can now annotate `@KafkaListener` methods (and classes, and `@KafkaHandler`
 If the method returns a result, it is forwarded to the specified topic.
 See <<annotation-send-to>> for more information.
 
-===== Message Listeners
+==== Message Listeners
 
 Message listeners can now be aware of the `Consumer` object.
 See <<message-listeners>> for more information.
 
-===== ConsumerAwareRebalanceListener
+==== ConsumerAwareRebalanceListener
 
 Rebalance listeners can now access the `Consumer` object during rebalance notifications.
 See <<rebalance-listeners>> for more information.
 
-===== @EmbeddedKafka Annotation
+==== @EmbeddedKafka Annotation
 
 For convenience a test class level `@EmbeddedKafka` annotation is provided with the purpose to register `KafkaEmbedded` as a bean.
 See <<testing>> for more information.
+
+==== Kerberos Configuration
+
+Support for configuring Kerberos is now provided.
+See <<Kerberos>> for more information.
```

Comment:
I don't merge yet.
I'm just fine with the current code.
Waiting for your confirmation about that `0.9` issue around system property.
Anyway we don't need to mention such a version in those comments since this code is already for higher versions.